SU-HSDO/suhumsci
# feat (STN-694): Merge Refined Padding for hb-raised-cards on Colorful

# [STN-694](https://sparkbox.atlassian.net/browse/STN-694) READY FOR REVIEW

## Summary
Merge the hb-raised-cards padding refinement for Colorful tested via the experimental feature.

### References
Original JIRA ticket: [STN-540](https://sparkbox.atlassian.net/browse/STN-540)
Original PR: #744

## Need Review By (Date)
3/5

## Urgency
low

## Steps to Test
1. In your CLI, run `npm run test` and confirm all tests pass.
2. Set your theme to Colorful and make sure the experimental features toggle is not activated.
- [x] Go to http://sparkbox-sandbox.suhumsci.loc/qa/qa-raised-postcards. Confirm the refined (reduced) padding on raised cards displays. See screenshots below for before and after examples.
- [x] Check the description for the experimental feature theme setting. Confirm that any reference to the padding refinement on Colorful has been removed.

### Before
![before](https://user-images.githubusercontent.com/12678977/109872803-ea8d2c80-7c3a-11eb-94d9-1a5d3d481078.png)

### After
![after](https://user-images.githubusercontent.com/12678977/109872902-098bbe80-7c3b-11eb-8547-b6dbdf67aad8.png)
